docs/source/en/api/pipelines/open_muse.md

```diff
@@ -0,0 +1,51 @@
+<!--Copyright 2023 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# Open MUSE
+
+Open MUSE is a light weight text to image model developed by the diffusers team at Hugging Face. Contributors include Suraj Patil, William Berman, and Patrick von Platen. Many thanks to community contributors <TODO community contributors?>, ... and Robin Rombach, <TODO advice people?> for advice.
+
+Open MUSE is a masked transformer model trained on LAION-5B and LAION-5B aesthetic subsets. There are two checkpoints, one for 256x256 and one for 512x512 image resolution.
+
+See the model card <TODO link> for more information on training procedure and data used.
```

Comment:
Should make it clear that the current release is a research release? Or making it clear in just the model card is fine?

Reply:
This q is for @patil-suraj @patrickvonplaten

Reply:
model card is good enough IMO